pyvlx/scene.py

```diff
@@ -39,7 +39,9 @@ async def run(self, wait_for_completion=True):
 
     def __str__(self):
         """Return object as readable string."""
-        return '<Scene name="{0}" ' 'id="{1}" />'.format(self.name, self.scene_id)
+        return '<{} name="{}" id="{}"/>'.format(
```

Comment:
What about using fancy python format strings here? (I would also be fine if we do this in a separate step)

Reply:
hope, i get you right.
it was inconsistent with single or double quotes (see eg. pyvlx/api/frames/frame_set_node_name.py)
i added type(self).__name__ is because i am lazy and dont want to type too much

seperate steps is maybe the main thing in this pr. too many things together

Reply:
Yes, fully agree. It is a good improvement. I just thought of using:

`return f'<{type(self).__name__} name="{self.name}" id="{self.scene_id}"/>'`

But im not sure if this is an improvement...

Reply:
nah. this is a mess. i should be even more strict
`         return '<{0} name="{1}" id="{2}"/>'.format( ...`